repository/u.json

```diff
@@ -22,11 +22,15 @@
 			]
 		},
 		{
-			"details": "https://github.com/frankban/UbuntuPaste",
+			"details": "https://github.com/3v1n0/UbuntuPaste",
 			"releases": [
 				{
 					"sublime_text": "<3000",
-					"branch": "master"
+					"tags": "2."
```

Comment:
You need to add a prefix. Most users who utilize this feature have a prefix of something like `st2-`. Then their tags would be `st2-2.0.0`, `st2-2.0.1`, etc. Users will see version `2.0.0` installed, as the prefix is only used to figured out which tags are relevant for the channel.